Fixup tecknet reference imagery

PeterJCLaw opened the pull request:
This renames each of the tecknet sample images in an attempt to match the labels to their actual images, mostly without adjusting the set of available labels (I think there's one removal and one addition). It then bundles models to match, though unfortunately the models still aren't particularly great.

This needs review from someone who was present when the images were captured as the label matching is based on assumptions about what distances are actually present in the images that might not be valid.

PeterJCLaw committed:
Equalise levels of prescision

PeterJCLaw committed:
Correct labelling of the tecknet 10cm calibrations

At some point it looks like the labels got mixed up. Assuming that
the set of labels was correct, I believe that this applies the
labels to the right images. Certainly this now ensures that the
sequences of offsets at each distance is in the right order.

PeterJCLaw committed:
Equalise levels of prescision

PeterJCLaw committed:
Correct labelling of the tecknet 25cm calibrations

At some point it looks like the labels got mixed up. Assuming that
the set of labels was correct, I believe that this applies the
labels to the right images. Certainly this now ensures that the
sequences of offsets at each distance is in the right order.

Unfortunately this also removes a number of images which were not
only mis-labeled, but were also duplicates of other (also
mis-labeled) images. I've left the related (now missing) entries
in the files.yaml as I'll try to back-fill them with the proper
images, if I can find them. If not, I'll remove them.
Either way this commit deliberately records the name fixes separately
from the restoration of the missing images.

PeterJCLaw committed:
Comment out the missing files

PeterJCLaw committed:
Update calibration for tecknet 25cm images

This adds in most of the missing images by copying them from #22,
though we still don't appear to have a valid source for one of
them (which is now removed).

PeterJCLaw committed:
Remove obsolete unused tecknet model

RealOrangeOne commented:
GitHub seems to detect some of these as new images, and some as renames, which is _odd_. 

Below is the list of marker locations captured on the night:
> 1m
0.7m
1m0.1l
1m0.2l
1m0.1r
1m0.2r
1.5m
1.5m0.1r
1.5m0.2r
1.5m0.3r
1.5m0.4r
1.5m0.1l
1.5m0.2l
1.5m0.3l
1.5m0.4l
2m
2m0.1l
2m 0.2l 
2m0.3l
2m 0.4l
2m 0.5l
2m0.6l
2m0.1r
2m 0.2r 
2m0.3r
2m0.4r
2m0.5r
2m0.6r
2.5m 
2.5m0.2l
2.5m 0.4l 
2.5m 0.6l
2.5m 0.8l
2.5m 0.2r 
2.5m 0.4r
2.5m 0.6r
2.5m 0.8r
3m 
3m 0.2l
3m 0.4l
3m 0.6l
3m 0.8l
3m1l
3m0.2r
3m0.4r
3m0.6r
3m0.8r
3m1r
4m
4m0.4l
4m0.8l
4m1.2l
4m0.4r
4m0.8r
4m1.2r
5m
5m0.5l
5m1l
5m1.5l
5m0.5r
5m1r
5m1.5r

PeterJCLaw commented:
I'm not sure why github is showing quite as many new files as it is, I suspect it doesn't like move chains like "a->b, b->c, c->a" which this does a lot of.

There are about six new files here, all in the 25cm size though.

Adimote reviewed (approved):
It can't be worse than what we currently have

PeterJCLaw committed:
Merge branch 'master' into fixup-tecknet-references

PeterJCLaw committed:
Add marker sizes to the tecknet calibrations

This was done manually as follows rather than by re-training:
``` python
>>> data = pickle.load(lzma.open('sb_vision/tecknet_10cm.pkl.xz'))
>>> data['marker_size'] = 0.1
>>> pickle.dump(data, lzma.open('sb_vision/tecknet_10cm.pkl.xz', mode='wb'))
>>> data = pickle.load(lzma.open('sb_vision/tecknet_25cm.pkl.xz'))
>>> data['marker_size'] = 0.25
>>> pickle.dump(data, lzma.open('sb_vision/tecknet_25cm.pkl.xz', mode='wb'))
```
